### PR TITLE
Implement API to get user's tickets

### PR DESCRIPTION
**Pull request by estyxx**
## Why

We need the get attendee's tickets in order to display them in the users profile.

### CODE REVIEW

File: pretix_extended_api/views/tickets.py [hunk 1]
```diff
@@ -46,3 +51,43 @@ def attendee_has_ticket(self, request, **kwargs):
 
         qs = qs.filter(events_filter)
         return Response({"user_has_admission_ticket": qs.exists()})
+
+    @scopes_disabled()
```
Comment (marcoacierno) [inline pretix_extended_api/views/tickets.py]:
```suggestion
```

I don't think we should disable the scopes check here

File: pretix_extended_api/views/tickets.py [hunk 2]
```diff
@@ -46,3 +51,43 @@ def attendee_has_ticket(self, request, **kwargs):
...
+                    {
+                        "question_id": a.question.id,
+                        "answer": a.answer,
+                        "options": [
+                            {"identifier": o.identifier, "answer": o.answer.data}
+                            for o in a.question.options.all()
+                        ],
+                        "question": a.question.question.data,
+                    }
+                    for a in order_position.answers.all()
```
Comment (marcoacierno) [inline pretix_extended_api/views/tickets.py]:
```suggestion
                    for answer in order_position.answers.all()
```

I think we should use more meaningful names than `a` and `o`

File: tests/views/tickets/test_attendee_tickets.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            "attendee_email": "marco@noticket.it",
+        },
+        format="json",
+    )
+
+    assert resp.status_code == 200
+    assert resp.data == []
+
+
+def test_email_has_tickets_with_multiple_events_different_organizers(
```
Comment (marcoacierno) [inline tests/views/tickets/test_attendee_tickets.py]: I don't think this test is useful here

in the other API we used to check tickets across multiple events

File: pretix_extended_api/views/tickets.py [hunk 4]
```diff
@@ -46,3 +51,43 @@ def attendee_has_ticket(self, request, **kwargs):
...
+    @action(url_path="attendee-tickets", detail=False, methods=["get"])
+    def attendee_tickets(self, request, *args, **kwargs):
+        check_permission(request, "can_view_orders")
+
+        serializer = AttendeeTicketBodySerializer(data=request.query_params)
+        serializer.is_valid(True)
+
+        attendee_email = serializer.data["attendee_email"]
+
+        qs = OrderPosition.objects.filter(
```
Comment (marcoacierno) [inline pretix_extended_api/views/tickets.py:67]: we should filter for event here, we do not want all the tickets of the user across all the events

Comment (marcoacierno) [reply pretix_extended_api/views/tickets.py:67]: you should have the organiser/event in the URL

File: tests/views/tickets/test_attendee_tickets.py (hunk 3 above)
Comment (estyxx) [reply tests/views/tickets/test_attendee_tickets.py]: I added a filter by events

File: pretix_extended_api/views/tickets.py (hunk 4 above)
Comment (estyxx) [reply pretix_extended_api/views/tickets.py:67]: I added a filter by events

**Review by marcoacierno — APPROVED** [review]

**Pull request merged**